generators/path_generator

```diff
@@ -80,8 +80,19 @@ for cfg in glob.glob(os.path.join(conf_dir, 'generators', 'meta-path',
                 if blacklist.count(basename) > 0:
                     print("Skipping blacklisted file: {}".format(f))
                     continue
-
+                # check if this file is in folder inside test directory
+                folder_path = os.path.dirname(
+                    f.replace(
+                        os.path.commonprefix(
+                            [
+                                f,
+                                os.path.abspath(
+                                    os.path.join(
+                                        third_party_dir, *path, match))
+                            ]), "")).replace("/", "_")
```

Comment:
wow, nice formatting. I see that the checker agrees with this, but maybe we could do something about it?